charts/common/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: common
 description: Function library for k8s-at-home charts
 type: library
-version: 2.3.0
+version: 2.3.1
```

Comment:
This can stay at 2.3.0 since version 2.3.0 is not yet released

Reply:
I had it liked that  first also but was getting an error in the CI check, so I bumped it.